packages/calcite-components/src/components/list-item/list-item.scss

```diff
@@ -14,6 +14,16 @@
 
 @include disabled();
 
+.indent {
```

Comment:
indent isn't a valid class on the list-item. Can this part be reverted? It should be on `.nested-container`. I'm guessing this was copied from a previous release.